src/mixins/filing-template-mixin.ts

```diff
@@ -139,6 +147,7 @@ export default class FilingTemplateMixin extends Vue {
     })
 
     // Date check to improve UX and work around default effectiveDate set by backend.
+    // TODO: fix types (should be Date)
```

Comment:
I noticed that the following variables are strings, while the store property is a Date. This should be resolved before we deploy again (which is why I made this a TODO instead of a FUTURE).
Any takers?